dietstory-routing/dietstory-routing/settings.py

```diff
@@ -77,6 +77,26 @@
     },
 ]
 
+LOGGING = {
+    'version': 1,
+    'disable_existing_loggers': False,
+    'handlers': {
+        'console': {
+            'class': 'logging.StreamHandler',
+        },
+    },
+    'loggers': {
+        'default-logger': {
+            'handlers': ['console'],
+            'level': os.getenv('DJANGO_LOG_LEVEL', 'INFO'),
+        },
+        'dev-logger': {
+            'handlers': ['console'],
+            'level': os.getenv('DJANGO_LOG_LEVEL', 'DEBUG')
```

Comment:
We should move the log level setting (ie `os.getenv('DJANGO_LOG_LEVEL', 'DEBUG')`) to the config file.

Reply:
Make sure to update the secret_template.json when introducing the new setting.

Reply:
Moved to config file and updated the template.